connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -198,6 +202,7 @@
     private final CookieStore cookieStore;
     private final boolean preemptiveBasicAuth;
     private final RequestConfig requestConfig;
+    private final Lookup<CookieSpecProvider> cookiespecRegistry;
```

Comment:
I don't know if "cookieSpecRegistry" is the correct came case form? Given the placement of underscores in COOKIE_SPEC_REGISTRY.